pom.xml

```diff
@@ -45,6 +45,7 @@
     <modules>
         <module>components</module>
         <module>themes</module>
+        <module>tests</module>
```

Comment:
if this is not used for testing purpose (i.e writing test artifacts and samples) then this module should be names as samples